README.md

```diff
@@ -24,7 +24,7 @@ inherit_from:
 
 ## Updating This Gem
 
-After making changes to the rules in the [default.yml](default.yml) file, be sure that the version
+After making changes to the rules in the [optimum_energy_rubocop.yml.yml](optimum_energy_rubocop.yml.yml) file, be sure that the version
```

Comment:
I think this line is a bit long now. Can you rewrap it and the ones below at 100 characters?

Reply:
Fixed 👍